### PR TITLE
Moving /info section to HTTPS

### PR DESCRIPTION
**Pull request by gklopper**
links to pages in secure sections will now be protocol relative.

Canonical tags for pages in HTTPS sections will now be HTTPS.

Also includes some football data files from tests

### CODE REVIEW

**Comment by OliverJAsh** [comment]
Protocol relative URLs are considered an anti-pattern:

> If the asset you need is available on SSL, then always use the https:// asset.

http://www.paulirish.com/2010/the-protocol-relative-url/

**Comment by gklopper** [comment]
@OliverJAsh that is the end goal. During this testing phase both HTTP and HTTPS need to co-exist and you need to stick on the one you showed up on.

We have however actually moved assets.guim.co.uk and api.nextgen.gaurdianapps.co.uk to full HTTPS

File: Gemfile.lock [hunk 1]
```diff
@@ -19,6 +19,3 @@ DEPENDENCIES
   hologram (= 1.2.0)
   sass (~> 3.3.14)
   scss-lint (= 0.26.2)
-
```
Comment (OliverJAsh) [inline Gemfile.lock]: @rich-nguyen

Comment (gklopper) [inline Gemfile.lock]: I hate that this auto updates

Comment (johnduffell) [inline Gemfile.lock]: source control is for our code, I too don't like checking in generated code/stuff we didn't write ourselves

File: static/src/systemjs-config.js [hunk 2]
```diff
@@ -132,7 +132,7 @@ System.config({
     "npm:assert@1.3.0": {
       "util": "npm:util@0.10.3"
     },
-    "npm:babel-runtime@5.6.15": {
```
Comment (johnduffell) [inline static/src/systemjs-config.js]: you might get conflicts on this, Oli checked in .18 to his PR

**Comment by johnduffell** [comment]
looks good :+1:

**Pull request merged**